server/events/event_parser.go

```diff
@@ -939,7 +939,12 @@ func (e *EventParser) ParseAzureDevopsRepo(adRepo *azuredevops.GitRepository) (m
 		host = "dev.azure.com"
 	}
 
-	cloneURL := fmt.Sprintf("https://%s/%s/%s/_git/%s", host, owner, project, repo)
+	cloneURL := ""
+	if strings.Contains(host, "visualstudio.com") {
+		cloneURL = fmt.Sprintf("https://%s/%s/_git/%s", host, project, repo)
+	} else {
+		cloneURL = fmt.Sprintf("https://%s/%s/%s/_git/%s", host, owner, project, repo)
```

Comment:
Would you mind updating the `TestParseAzureDevopsRepo` test to check both cloneURL conditions?

https://github.com/runatlantis/atlantis/blob/6a7f79e749a1604f54378e4920cf83c2888399ec/server/events/event_parser_test.go#L1203

Reply:
I think I've added this. Let me know if I need to make any changes (am very new to Go.